Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx

```diff
@@ -240,9 +240,10 @@ export const editorSteps: { [key in ManifestEditorSteps]: EditorStep } = {
     content: AddCallers,
     subText: () =>
       formatMessage(
-        'To ensure a secure connection, provide the App ID of the bots that can connect to your skill.  If you don’t have this information, you can also add this information in Skill Configuration. Learn more.'
+        'To ensure a secure connection, provide the App ID of the bots that can connect to your skill.  If you don’t have this information, you can also add this information in Skill Configuration.'
       ),
     title: () => formatMessage('Which bots can connect to this skill?'),
+    helpLink: 'https://docs.microsoft.com/en-us/composer/how-to-publish-skill?tabs=v2x',
```

Comment:
Make this an aka.ms link. Also, if it’s used multiple times, move it into a variable.

Reply:
Fixed it.